iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -187,6 +188,7 @@ public RatisConsensus(ConsensusConfig config, IStateMachine.Registry registry)
         RaftServer.newBuilder()
             .setServerId(myself.getId())
             .setProperties(properties)
+            .setOption(RaftStorage.StartupOption.RECOVER)
```

Comment:
Incompatible change. We must add RaftStorage.StartupOption.RECOVER here. Same as we need to set format to false when adding a new group.